src/components/views/rooms/EventTile.js

```diff
@@ -1,7 +1,7 @@
 /*
 Copyright 2015, 2016 OpenMarket Ltd
 Copyright 2017 New Vector Ltd
-Copyright 2019 The Matrix.org Foundation C.I.C.
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
```

Comment:
Extreme nitpick: can we move this line to below Michael so the years are vaguely ordered?